### PR TITLE
Format w/ prettier

### PR DESCRIPTION
**Pull request by grod220**
Closes https://github.com/penumbra-zone/dex-explorer/issues/82

RIP folks with in-progress PRs...

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,10 +26,11 @@ jobs:
       - uses: pnpm/action-setup@v4
       - uses: buildjet/setup-node@v4
         with:
-          node-version: "18"
-          cache: "pnpm"
+          node-version: '18'
+          cache: 'pnpm'
       - run: pnpm install --frozen-lockfile
       - run: pnpm lint:strict
+      - run: pnpm lint:prettier
```
Comment (grod220) [inline .github/workflows/ci.yml:33]: Added CI/CD check

File: package.json [hunk 2]
```diff
@@ -12,10 +12,11 @@
     "build": "next build",
     "dev": "next dev",
     "start": "next start",
+    "format": "prettier --write .",
     "lint": "eslint src",
     "lint:fix": "eslint src --fix",
     "lint:strict": "tsc --noEmit && pnpm lint --max-warnings 0",
-    "lint:strict2": "tsc --noEmit && pnpm lint --max-warnings 0"
+    "lint:prettier": "prettier --check ."
```
Comment (grod220) [inline package.json:19]: package.json scripts that mirror penumbra-zone/web

**Comment by conorsch** [comment]
FYI I snuck in some small CI changes in #90 & #91, towards #72. Mentioning here in case the YAML formatting throws an error once rebased. Leaving said rebasing to you, but holler at me if I broke anything and you want more eyes on it.

**Comment by grod220** [comment]
No worries, a mass formatting PR is very little work in getting rebased to the latest commit, It's basically just re-running formatter 👍

**Review by JasonMHasperhoven — APPROVED** [review]

**Pull request merged**